### DISCUSSION TITLE
Openssl 3.0.13 cant build

### DISCUSSION THREAD

**Discussion by knightwing1b** [openssl/openssl#23736] [Q&A] [answered]
Trying to build openssl 3.0.13 on Windows.
I have install 3.0.8 to 3.0.12 with no problem but this time when I run

Perl configure vc-win64a on same machine and same perl etc

I get this error

E:\myfolder\openssl-3.0.13\crypto\provider:
No such file or directory at configure line 3413

No change in my system.

**Comment by joshcooper** [comment, marked as answer]
This is fixed in https://github.com/openssl/openssl/commit/7b3eda56d7891aceef91867de64f24b20e3db212
